vaadin-date-picker-flow/pom.xml

```
@@ -100,7 +100,7 @@
         <dependency>
             <groupId>org.webjars.bowergithub.vaadin</groupId>
             <artifactId>vaadin-text-field</artifactId>
-            <version>2.4.8</version>
+            <version>2.6.0-alpha4</version>
```

Text-field is stable.
```suggestion
            <version>2.6.0</version>
```